engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/semantic/NestedPatternCompilationBenchmarks.java

```diff
@@ -38,7 +39,7 @@ public class NestedPatternCompilationBenchmarks {
     public void initializeBenchmark(BenchmarkParams params) throws Exception {
         ctx = Context.newBuilder()
                 .allowExperimentalOptions(true)
-                .allowIO(true)
+                .allowIO(IOAccess.ALL)
```

Comment:
I hope this is just a deprecation and the original method still remains in the API. Does it?

Reply:
Yes, this is just a deprecation, that was very straightforward to remove. The original method still remains in the API, but for how long?